tools/pd-ctl/pdctl/command/unsafe_command.go

```diff
@@ -67,21 +67,41 @@ func NewRemoveFailedStoresHistoryCommand() *cobra.Command {
 
 func removeFailedStoresCommandFunc(cmd *cobra.Command, args []string) {
 	prefix := fmt.Sprintf("%s/remove-failed-stores", unsafePrefix)
-	if len(args) != 1 {
+	if len(args) != 1 || len(args) != 2 {
```

Comment:
With `remove-failed-stores 1,2,3 --timeout 60`, seems the len(args) should be 3?